src/app/api/tasks/tasks.service.ts

```diff
@@ -532,4 +538,15 @@ export class TasksService extends BaseService {
     // --- Handle task moved to complete
     await notificationService.markAsReadForAllRecipients(updatedTask)
   }
+
+  async getSignedUrl(filePath: string) {
+    const supabase = new SupabaseService()
+    const { data, error } = await supabase.supabase.storage.from(supabaseBucket).createSignedUrl(filePath, 60 * 60)
```

Comment:
let's avoid magic numbers. can we create a constant for these so that it is understandable.